DabTester.py

```diff
@@ -19,14 +19,15 @@ def execute_cmd(self,device_id,dab_dab_request_body_topic,dab_request_body="{}")
             return 1
     
     def Execute_Test_Case(self, device_id, test_case):
-        (dab_dab_request_body_topic, dab_request_body, validate_output_function, expected_response_code)=test_case
```

Comment:
Why not expand the scope of validate_output_function and call it validate_response.

Reply:
What do you mean here? You mean I call validate_output_function with validate_reaponse?

Reply:
I meant merge all validations into one function thats called validate_reaponse or any other name you like to use.

Reply:
Oh, I see, that makes sense. I will try to refactor that.

Reply:
I made another commit to reflect this, let me know if that's what you mean!